nflow-tests/src/test/java/io/nflow/tests/MaintenanceWorkflowTest.java

```diff
@@ -76,7 +76,7 @@ public void createWorkflows() {
   @Test
   @Order(4)
   public void waitForCleanup() throws InterruptedException {
-    SECONDS.sleep(6);
+    SECONDS.sleep(8);
```

Comment:
This could be removed if you change verifyThatCleanupOccurred so that it tries to get the NotFoundException and if not sleeps for one second and tries again (until finally timing out after 10 seconds for example)